src/redprl/machine.sml

```diff
@@ -168,6 +168,19 @@ struct
          S.CUT
            @@ (coe `$ [([u],[]) \ S.HOLE, ([],[]) \ S.% m], a)
            <: env
+     | O.POLY (O.COM (dir as (r, r'), eqs)) `$ (([u],_) \ a) :: (_ \ cap) :: tubes <: env =>
+         let
+           fun coe v m = Syn.intoCoe (v, r') ((u, a), m)
```

Comment:
It may be clearer to write something like `s` instead of `v` here, because we usually use `u,v` as schematic variables that range over dimension variables only.